src/view/cards/wdr/wdr_content_card.py

```diff
@@ -15,10 +17,11 @@ class WDRContentCard:
 
     domain_mapping = {"wdr.de": "https://www1.wdr.de"}
 
-    def __init__(self, config):
+    def __init__(self, config, reco_explorer_app_instance:RecoExplorerApp=None):
         self.config = config
+        self.reco_explorer_app_instance = reco_explorer_app_instance
 
-    def draw(self, content_dto: WDRContentItemDto, card):
+    def draw(self, content_dto: WDRContentItemDto, card, button:pn.widgets.Button=None):
```

Comment:
hier auch optional type:
```suggestion
    def draw(self, content_dto: WDRContentItemDto, card, button: pn.widgets.Button | None=None):
```